src/test/runner.ts

```diff
@@ -254,6 +255,23 @@ function filterOnly(suite: Suite) {
   return false;
 }
 
+function filterTestFileByLine(suite: Suite, focusedTestFileLines: FilePatternFilter[]) {
```

Comment:
I see a lot of similarity to `filterOnly`. Let's maybe generalize to `filterSuite(suite, Spec => boolean)`?

Reply:
This is a great idea.